cmake/Modules/FindQwt.cmake

```diff
@@ -51,13 +51,13 @@ if(QWT_INCLUDE_DIRS)
   set(QWT_VERSION "No Version")
   string(REGEX MATCH "[0-9]+.[0-9]+.[0-9]+" QWT_VERSION ${QWT_STRING_VERSION})
   string(COMPARE LESS ${QWT_VERSION} "5.2.0" QWT_WRONG_VERSION)
-  string(COMPARE GREATER ${QWT_VERSION} "6.2.0" QWT_WRONG_VERSION)
```

Comment:
Why wouldn't this be valid if we are now allowing QWT 6.2 - seems the check would have been previously too permissive, but is now correct

Reply:
After thinking about this some more, it should be left as it is in `master`. The check before was letting in a version that was unreleased (and incompatible). I've reverted the change.